micronota/parsers/tests/test_cmscan.py

```diff
@@ -0,0 +1,187 @@
+            'MODEL_NAME': 'LSU_rRNA_bacteria',
+            'MODEL_ACCESSION': 'RF02541',
+            'SEQUENCE_NAME': 'gi|15829254|ref|NC_002695.1|',
+            'SEQUENCE_ACCESSION': '-',
+            'TYPE_OF_MODEL': 'cm',
+            'MODEL_START_POSITION': '1',
+            'MODEL_END_POSITION': '2925',
+            'STRAND': '+',
+            'TRUNCATED': 'no',
+            'PASS': '1',
+            'GC_CONTENT': '0.53',
+            'BIAS': '45.4',
+            'BITSCORE': '2889.8',
+            'EVALUE': '0',
+            'INC': '!',
+            'DESCRIPTION': '-'
+        }, {
+            'MODEL_NAME': 'SSU_rRNA_microsporidia',
+            'MODEL_ACCESSION': 'RF02542',
+            'SEQUENCE_NAME': 'gi|15829254|ref|NC_002695.1|',
+            'SEQUENCE_ACCESSION': '-',
+            'TYPE_OF_MODEL': 'cm',
+            'MODEL_START_POSITION': '1',
+            'MODEL_END_POSITION': '1312',
+            'STRAND': '+',
+            'TRUNCATED': 'no',
+            'PASS': '1',
+            'GC_CONTENT': '0.55',
+            'BIAS': '14.5',
+            'BITSCORE': '733.3',
+            'EVALUE': '5.8e-220',
+            'INC': '!',
+            'DESCRIPTION': '-'
+        }, {
+            'MODEL_NAME': 'IS061',
+            'MODEL_ACCESSION': 'RF00115',
+            'SEQUENCE_NAME': 'gi|secondSeq|',
+            'SEQUENCE_ACCESSION': '-',
+            'TYPE_OF_MODEL': 'cm',
+            'MODEL_START_POSITION': '1',
+            'MODEL_END_POSITION': '180',
+            'STRAND': '-',
+            'TRUNCATED': 'no',
+            'PASS': '1',
+            'GC_CONTENT': '0.44',
+            'BIAS': '0.0',
+            'BITSCORE': '232.1',
+            'EVALUE': '1.5e-42',
+            'INC': '!',
+            'DESCRIPTION': '-'
+        }, {
+            'MODEL_NAME': 'SIB_RNA',
+            'MODEL_ACCESSION': 'RF00113',
+            'SEQUENCE_NAME': 'gi|secondSeq|',
+            'SEQUENCE_ACCESSION': '-',
+            'TYPE_OF_MODEL': 'cm',
+            'MODEL_START_POSITION': '1',
+            'MODEL_END_POSITION': '147',
+            'STRAND': '+',
+            'TRUNCATED': 'no',
+            'PASS': '1',
+            'GC_CONTENT': '0.40',
+            'BIAS': '0.0',
+            'BITSCORE': '136.7',
+            'EVALUE': '4e-27',
+            'INC': '!',
+            'DESCRIPTION': '-'
+        }]
+        self.intervals = [
+            [(4977823, 4980727)],
+            [(4831659, 4833190)],
+            [(1915102, 1915281)],
+            [(3794337, 3794487)]
+        ]
+
+        self.seq1 = IntervalMetadata(features={
+            Feature(**self.subsequences[0]): self.intervals[0],
+            Feature(**self.subsequences[1]): self.intervals[1]
+        })
+        self.seq2 = IntervalMetadata(features={
+            Feature(**self.subsequences[2]): self.intervals[2],
+            Feature(**self.subsequences[3]): self.intervals[3]
+        })
+
+
+class ReaderTests(CmscanIOTests):
+    def test_cmscan_to_metadata(self):
```

Comment:
Also worth looking into [`setUp`](https://docs.python.org/2.7/library/unittest.html#unittest.TestCase.setUp).  We use it in our other tests.